docs/installation/ansible-galaxy.rst

```diff
@@ -0,0 +1,22 @@
+Pulp 3 Ansible Galaxy Installation
+==================================
+
+1. In order to be able to run playbook, ensure python3, ansible( >2.2) and libselinux-python are installed:
```

Comment:
libselinux-python should be installed on the target machines:
http://docs.ansible.com/ansible/latest/intro_installation.html#managed-node-requirements